serverlessworkflow/modules/ROOT/pages/core/understanding-workflow-error-handling.adoc

```diff
@@ -7,31 +7,43 @@ Understanding workflow error handling
 // links
 :java_exception_url: https://docs.oracle.com/javase/tutorial/essential/exceptions/index.html
 
-This guide explains how to handle errors that might occur while executing workflow states. Although a non-desirable situation, Workflow states might fail as part of their interaction with external systems. For these situations, the Serverless Workflow specification provides an link:{spec_doc_url}#workflow-error-handling[error handling] mechanism, allowing users to deal with these rainy day circumstances.
+This document describes how you can handle the errors that might occur when you execute your Serverless Workflow states. 
```

Comment:
Not sure if "when you execute your Serverless Workflow states" passes the right message. It's not the user who executes the states; it's the engine. Too bad if left as "when the Serverless Workflow states are executed"? I know that's passive voice, but it makes more sense to me.